src/MentionsTextInput.js

```diff
@@ -578,7 +578,7 @@ export default class MentionsTextInput extends Component {
           maxLength={this.props.maxLength ? this.props.maxLength : Number.MAX_SAFE_INTEGER}
           enablesReturnKeyAutomatically={this.props.enablesReturnKeyAutomatically ? this.props.enablesReturnKeyAutomatically : false}
           underlineColorAndroid={this.props.underlineColorAndroid ? this.props.underlineColorAndroid : 'black'}
-          editable={this.props.editable ? this.props.editable : true}
+          editable={this.props.editable}
```

Comment:
why are we doing this?

Reply:
Because muted users shouldn't be able to edit the text field, will this cause other problems?

Reply:
It shouldn't. The cases where a user should not be able to edit the text input is being handled in `ChatInputToolbar`. They aren't able to edit it, I tried hitting enter, and that doesn't do anything either. The only problem I kind of see is, you are able to select the text, and select all and copy it